src/gui/search/searchjobwidget.h

```diff
@@ -83,8 +84,11 @@ class SearchJobWidget : public QWidget
     void cancelSearch();
 
     void downloadTorrents();
-    void openTorrentPages();
-    void copyTorrentURLs();
+    void openTorrentPages() const;
+    void copyTorrentURLs() const;
+    void copyTorrentDownloadLinks() const;
+    void copyTorrentNames() const;
+    void copyField(const SearchSortModel::SearchColumn column) const;
```

Comment:
Maybe declare it as `int column` and don't pollute outer world by "searchsortmodel.h"?
`const` should be removed from here in any case.

Reply:
OK, done.